website/static/js/institutionProjectSettings.js

```diff
@@ -2,105 +2,189 @@
-            Raven.captureMessage('Unable to add institution to this node', {
-                extra: {
-                    url: url,
-                    status: status,
-                    error: error
-                }
-            });
-        });
     };
     self.clearInst = function(item) {
-        bootbox.confirm({
-            title: 'Are you sure you want to remove institutional affiliation from your project?',
-            message: 'You are about to remove affiliation with ' + item.name + ' from this project. ' + item.name + ' branding will not longer appear on this project, and the project will not be discoverable on the ' + item.name + ' landing page.',
-            callback: function (confirmed) {
-                if (confirmed) {
-                    self._clearInst(item);
-                }
-            },
-            buttons:{
-                confirm:{
-                    label:'Remove affiliation'
-                }
-            }
-        });
+        self.needsWarning((self.userInstitutionsIds().indexOf(item.id) === -1));
+        self.isAddInstitution(false);
+        if (self.hasChildren()) {
+            self.modifyChildrenDialog(item);
+        }
+        else {
+            return self._modifyInst(item);
+        }
     };
-    self._clearInst = function(item) {
-        var url = data.apiV2Prefix + 'nodes/' + data.node.id + '/relationships/institutions/';
+
+    self._modifyInst = function(item) {
+        var url = data.apiV2Prefix + 'institutions/' + item.id + '/relationships/nodes/';
+        var ajaxJSONType = self.isAddInstitution() ? 'POST': 'DELETE';
+        var nodesToModify = [];
+        if (self.modifyChildren()) {
+            for (var node in self.nodesOriginal()) {
+                nodesToModify.push({'type': 'nodes', 'id': self.nodesOriginal()[node].id});
+            }
+        }
+        else {
+                nodesToModify.push({'type': 'nodes', 'id': self.nodeParent});
+        }
         return $osf.ajaxJSON(
-            'DELETE',
+            ajaxJSONType,
             url,
             {
                 isCors: true,
                 data: {
-                     'data': [{'type': 'institutions', 'id': item.id}]
+                     'data': nodesToModify
                 },
                 fields: {xhrFields: {withCredentials: true}}
             }
-        ).done(function (response) {
+        ).done(function () {
             var indexes = self.affiliatedInstitutions().map(function(each){return each.id;});
-            var removed = self.affiliatedInstitutions().splice(indexes.indexOf(item.id), 1)[0];
-            if ($.inArray(removed.id, self.userInstitutionsIds) >= 0){
-                self.availableInstitutions().push(removed);
+            if (self.isAddInstitution()) {
+                var index = indexes.indexOf(self.selectedInstitution());
```

Comment:
Can you check this logic? This seems a likely culprit for this bug: https://github.com/CenterForOpenScience/osf.io/pull/5726#issuecomment-221649965